starlite/testing/test_client/client.py

```diff
@@ -47,11 +54,25 @@
     )
     from typing_extensions import Literal
 
-    from starlite.middleware.session import SessionCookieConfig
+    from starlite.middleware.session.base import BaseBackendConfig, BaseSessionBackend
     from starlite.testing.test_client.websocket_test_session import WebSocketTestSession
 
 
 T = TypeVar("T", bound=ASGIApp)
+AnySessionBackend = Union[CookieBackend, ServerSideBackend]
+AnySessionConfig = Union["ServerSideSessionConfig", "CookieBackendConfig"]
+
+
+def raise_for_unsupported_session_backend(backend: "BaseSessionBackend") -> None:
+    raise ImproperlyConfiguredException(f"Backend of type {type(backend)!r} is currently not supported")
+
+
+def warn_for_deprecated_session_interface() -> None:
+    warnings.warn(
+        "Accessing the session via this property is considered legacy and will be deprecated"
+        "in future versions. To access the session backend directly, use the session_backend attribute",
```

Comment:
```suggestion
        "To access the session backend directly, use the session_backend attribute",
```

Can you write this out of the docs? I mean this access method.

Reply:
What do you mean? Make this method undocumented?

Reply:
I mean, if it's deprecated either remove it from the documentation, or explicitly mark it as deprecated there.

Reply:
Ah, gotcha. Will do

Reply:
Done